homeassistant/components/light/hue.py

```diff
@@ -179,6 +179,7 @@ def update_lights():
                 new_lights.append(lights[light_id])
             else:
                 lights[light_id].info = info
+                lights[light_id].update_ha_state()
```

Comment:
Please use `schedule_update_ha_state()` instead. We shouldn't `use update_ha_state()` as it keeps things in sync.

Adding it here shouldn't make any extra impact, just uses some extra resources.

Entity component who is in charge of updating lights will work like this. Imagine we have lights A, B, C.

Update light A -> fetches new data and causes light A, B, C to write their state to state machine
Write state of light A to state machine
Update light B -> does nothing because of throttle
Write state of light B to state machine
Update light C -> does nothing because of throttle
Write state of light C to state machine

So now each light will now have their state written to the state machine twice. The state machine will filter out the states so it will become a no-op.